src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -61,8 +61,8 @@
     <Reference Include="Microsoft.Data.Services.Client, Version=5.8.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.WindowsAzure.Storage, Version=9.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
assuming this will be undone once the new SDK is released.

Reply:
Yes, I will update all the reference when the XSCL is release. Currently remove the “PublicKeyToken” only to make the private XSCL work.